modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsControllerEpics.scala

```diff
@@ -209,16 +209,32 @@ object TcsControllerEpics extends TcsController {
         setOiwfs, EpicsTcsConfig.oiwfs ^|-> GuiderConfig.detector)
     ).collect{ case Some(x) => x }
 
+
+  def tagIso[B, T]: Iso[B@@T, B] = Iso.apply[B@@T, B](x => x)(tag[T](_))
+
   def calcGuideOff(current: EpicsTcsConfig, demand: TcsConfig): TcsConfig = {
-    if(willMove(current, demand)) (TcsConfig.gds.modify(
```

Comment:
The main change here is that in the guide-off step, I should only turn thing off. Before I was returning the original demand, which may have some things turned on